CONFIGURATION.md

```diff
@@ -44,6 +47,12 @@ The AWS key for access for this S3 bucket
 The AWS retry window in seconds.
 #### s3.bucket.uri = ucentral-ap-firmware.s3.amazonaws.com
 The URI to the S3 bucket
+#### s3.endpointOverride = ""
```

Comment:
I added a description to CONFIGURATION.md

is there another place? or maybe my descriptions arent complete enought